### ISSUE TITLE
MSG_MINIMAL env var is not working

### ISSUE THREAD

**Issue by johnhiggs** [rtCamp/action-slack-notify#38]
`MSG_MINIMAL` setting to true env var is not working

**Comment by mrrobot47** [comment]
Please use tag `@master` to get this feature. This has not been released to a stable version yet.

Ref: https://github.com/rtCamp/action-slack-notify/issues/33#issuecomment-643248019

**Comment by johnhiggs** [comment]
apologies didn't realize that, thanks

**Comment by mrrobot47** [comment]
No problem :) In fact, thanks for bringing it to attention. 
Need to take care of this documentation issue, will update readme to contain `master` tag from now on as well as have this released soon.

**Comment by mrrobot47** [comment]
This is now in a stable released version [v2.1.0](https://github.com/rtCamp/action-slack-notify/releases/tag/v2.1.0)

You can use `@master` or `@v2.1.0` tag to use this feature:

```yml
      uses: rtCamp/action-slack-notify@master
```
or 
```yml
      uses: rtCamp/action-slack-notify@v2.1.0
```
